.github/actions/publish-tag-and-release/action.yml

```diff
@@ -0,0 +1,92 @@
+#################################################################################
+#  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+#  Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0.
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#################################################################################
+
+
+---
+name: "Publish Github release and tag"
+description: "Publish Github release and tag"
+inputs:
+  version:
+    required: true
+    description: "The version to be used in the tag and release publication"
+  token:
+    required: true
+    description: "Github token"
+  isLatest:
+    required: true
+    description: "Boolean that defines if a release is latest or not"
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v4
+    - name: Prepare Git Config
+      shell: bash
+      run: |
+        # Prepare git env
+        git config user.name "eclipse-tractusx-bot"
+        git config user.email "tractusx-bot@eclipse.org"
+    - name: Create Release Tag
+      id: create_release_tag
+      shell: bash
+      run: |
+        # informative
+        git branch -a
+        git tag
+
+        # Create & push tag
+        git tag --force ${{ inputs.version }}
```

Comment:
using `--force` does not look really safe

Reply:
Agree. I overlooked this and figured it was needed at some point. Maybe to override existing tags? I don't think it's a good practice anyway.

Reply:
I would just remove that, the only advantage is that it speeds-up the process in case of re-launch (because of a failure) but it exposes the release to affect older releases.

Reply:
[3ea619b](https://github.com/eclipse-tractusx/tractusx-edc/pull/1543/commits/3ea619b7c4e94c36c9efbb977336e6dbd521b639)